en/docs/develop/advanced-development/applying-security-to-a-proxy-service.md

```diff
@@ -21,132 +21,12 @@ Follow the instructions below to create a **WS-Policy** resource in your integra
        **Create Options**: Select `From existing template`.       
        **Resource Name**: Enter a name for the resource.    
        **Template Type**: Select `WS-Policy`.    
+       **WS-Policy Type**: Select the WS Policy type. In this case select `Username Token`. Based on the policy type fill the popping up fields if required.  
        **Resource Path**: Provide the resource path where the resource should be saved.  
        [![Create New Resource]({{base_path}}/assets/img/integrate/apply-security/add-ws-policy/create-resource.png)]({{base_path}}/assets/img/integrate/apply-security/119130870/create-new-registry-resource.png)  
 6. The created policy file will be listed in the project explorer.  
     [![Resource in Project Explorer]({{base_path}}/assets/img/integrate/apply-security/add-ws-policy/resource-view.png)]({{base_path}}/assets/img/integrate/apply-security/119130870/119130883.png)
 
-    !!! Note
```

Comment:
Don't we need to have this note as currently the only way to edit a policy file is via the source view?

Reply:
+1. I modified the note, as we now support certain parameters though.